test/gtest/uct/uct_test.h

```diff
@@ -189,15 +189,15 @@ class uct_test : public testing::TestWithParam<const resource*>,
     virtual void init();
     virtual void cleanup();
     virtual void modify_config(const std::string& name, const std::string& value);
-    virtual void get_config(const std::string& name, std::string& value);
+    virtual bool get_config(const std::string& name, std::string& value);
```

Comment:
Not in this case... lack of value in the config should not fail the test and i do not think that catch an exception for fallback branch is a good usage of throw/catch mechanism 
(In general, this is a big debate about return code vs exceptions :) )